04-simulate-an-election/election.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+A = "a"
+B = "b"
+
+def elect_in_region(election, chances):
+    if election <= chances:
+        return A
+    else:
+        return B
+
+def simulate_a_single_election():
+    candidate_a_chances = {
+        "region1": 87,
+        "region2": 65,
+        "region3": 17
+    }
+    candidate_a_elected_count = 0
+    candidate_b_elected_count = 0
+
+    for region in candidate_a_chances:
+        random_result = int(random() * 100)
+        elected = elect_in_region(election=random_result, chances=candidate_a_chances[region])
+        if elected == A:
+            candidate_a_elected_count += 1
+        else:
+            candidate_b_elected_count += 1
+
+    if candidate_a_elected_count > candidate_b_elected_count:
+        return A
+    else:
+        return B
+
+def simulate_elections(elections_count):
+    candidate_a_elections_count = 0
+    candidate_b_elections_count = 0
+
+    for i in range(elections_count):
+        election_result = simulate_a_single_election()
+
+        if election_result == A:
+            candidate_a_elections_count += 1
+        else:
+            candidate_b_elections_count += 1
+
+    print("Candidate A won", candidate_a_elections_count, "times")
+    print("Candidate B won", candidate_b_elections_count, "times")
+
+    return {
+        "count": elections_count,
+        "a": candidate_a_elections_count,
+        "b": candidate_b_elections_count
+    }
+
+
+def calculate_win_probability(elections_count, candidate_winning_count):
+    return (float(candidate_winning_count) / float(elections_count)) * 100
```

Comment:
i don't understand how to do that because of the grouping and * 100 on that,  can you please explain?